beater/handlers.go

```diff
@@ -10,9 +10,9 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/hashicorp/golang-lru"
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
any particular reason to add this?

Reply:
I removed all imports after a merge conflict and by using `goimport` to automatically resolve imports afterwards, this was added automatically. Afaik if a package starts with `golang-` or `go-` this prefix is removed by default when using it.